src/run.js

```diff
@@ -123,7 +139,7 @@ const run = ({ testPath, config, globalConfig }) => {
               return resolve(
                 fail({
                   ...result,
-                  errorMessage: 'Not a valid OpenAPI schema.'
+                  errorMessage: 'Not a valider OpenAPI schema: ' + error.message
```

Comment:
Use a template string.

```suggestion
                  errorMessage: `Not a valider OpenAPI schema: ${error.message}`,
```